### PR TITLE
Fix Qemu CVE-2020-10702, CVE-2020-10761

### PR DESCRIPTION
**Pull request by henryli001**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `./.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Resolve CVE-2020-10702, CVE-2020-10761

###### Change Log  

- CVE-2020-10702
- CVE-2020-10761

###### Does this affect the toolchain?  

NO

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2020-10702
- https://nvd.nist.gov/vuln/detail/CVE-2020-10761

###### Test Methodology

- Local build

### CODE REVIEW

File: SPECS/qemu-kvm/qemu-kvm.spec [hunk 1]
```diff
@@ -62,6 +63,8 @@ This package provides a command line tool for manipulating disk images.
 %patch5 -p1
 %patch6 -p1
 %patch7 -p1
+%patch12 -p1
+%patch13 -p1
```
Comment (niontive) [inline SPECS/qemu-kvm/qemu-kvm.spec:67]: If it's not inconvenient, could you change this to %autosetup?

File: SPECS/qemu-kvm/qemu-kvm.spec [hunk 2]
```diff
@@ -122,36 +125,54 @@ chmod 755 %{buildroot}%{_bindir}/qemu
 %{_bindir}/qemu-nbd
 
 %changelog
+* Tue Oct 27 2020 Henry Li <lihl@microsoft.com> - 4.2.0-14
```
Comment (niontive) [inline SPECS/qemu-kvm/qemu-kvm.spec:128]: Nit: remove the "-" between "microsoft.com>" and "4.2.0-14".

Comment (thcrain-msft) [reply SPECS/qemu-kvm/qemu-kvm.spec:128]: @niontive This is actually our preferred style now, as enforced by the linter.

Comment (niontive) [reply SPECS/qemu-kvm/qemu-kvm.spec:128]: Gotcha- I was just considering the older changelog for this spec. Will note for the future!

**Review by niontive — APPROVED** [review]

**Pull request merged**